manager/base.py

```diff
@@ -147,20 +148,41 @@ def _parse_arguments(self, argv):
             self.raiseMany(400, errors, True)
         return retval
 
-    def _check_int_arg(self, key, dflt):
+    def _check_int_arg(self, key, dflt, zero_allowed=True):
         try:
             val = int(self.get_argument(key, dflt))
-            if val < 0:
+            if val < 0 or (val == 0 and not zero_allowed):
                 raise InvalidArgumentException("Requested %s out of range: %s" % (key, val))
             return val
         except ValueError:
             raise InvalidArgumentException("Invalid %s number: %s" % (key, self.get_argument(key, None)))
 
+    def _avail_list_arguments(self):
```

Comment:
Is extraction to this function really needed, it's not used anywhere else and IMO short enough to be inline.
But mileage may wary and I am OK with it being as it is.

Reply:
concur, removing